api/src/pcapi/routes/public/collective/endpoints/offers.py

```diff
@@ -480,6 +490,11 @@ def patch_collective_offer_public(
             },
             status_code=422,
         )
+    except (offers_exceptions.UnknownOfferSubCategory, offers_exceptions.SubcategoryNotEligibleForEducationalOffer):
+        raise ApiErrors(
+            errors={"subcategoryId": ["La subcategory est invalide."]},
```

Comment:
```suggestion
            errors={"subcategoryId": ["La sous-catégorie est invalide."]},
```

ou bien : 

```suggestion
            errors={"subcategoryId": ["La sous-catégorie n'existe pas ou n'est pas éligible pour les offres collectives.."]},
```

... comme tu le sens :)

Reply:
ha mince je l'avais oublié celui la ><